http/swagger.yml

```diff
@@ -2217,6 +2217,12 @@ paths:
               type: string
             required: true
             description: The ID of the dashboard to update.
+          - in: query
+            name: include
+            required: false
+            schema:
+              type: string
+            description: Includes the cell view properties in the response if set to `properties`
       responses:
           '200':
             description: Get a single dashboard
```

Comment:
Could you please also update the schema that may be returned? I can work with you on this if you wish.

Reply:
yeah sure, I could use your 2-cents on adding this to swagger